src/test/shared/sam/debugger/samDebugConfigProvider.test.ts

```diff
@@ -1153,6 +1205,126 @@ Globals:
 `
             )
         })
+
+        it('debugconfig with aws section', async () => {
```

Comment:
i do not think we should copy-paste the tests just to test 1 dimension. Each of the existing tests is already a complete test of inputs/outputs, so in order to test a new field, just add the field to one or more existing tests. There is no need to have a test for each field in the input config.